CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on Keep a Changelog, and this project adheres to Semantic Versioning.
 
+## [0.9.1 - 2024-09-14]
```

Comment:
Should this be 0.9.2 and another date? Or maybe the values are just waiting the final release date?

Reply:
Waiting for final release date :)